2024-12-30 18:23:29,107 - llmLogger - DEBUG - device = cuda:0
''2024-12-30 18:26:24' | llmLogger | INFO     | device = cuda:0'
''2024-12-30 18:27:39' | llmLogger | INFO     | ColQwen2ForRAG.py:78 - device = cuda:0'
''2024-12-30 18:30:33' | llmLogger | INFO     | ColQwen2ForRAG.py:78 - device = cuda:0'
''2024-12-30 18:30:33' | llmLogger | INFO     | ColQwen2ForRAG.py:83 - LoRA config загружена успешно.'
''2024-12-30 18:30:35' | llmLogger | INFO     | ColQwen2ForRAG.py:86 - Процессор для извлечения загружен успешно.'
''2024-12-30 18:30:36' | llmLogger | INFO     | ColQwen2ForRAG.py:89 - Процессор для генерации загружен успешно.'
''2024-12-30 18:34:49' | llmLogger | INFO     | ColQwen2ForRAG.py:88 - device = cuda:0'
''2024-12-30 18:34:49' | llmLogger | INFO     | ColQwen2ForRAG.py:93 - LoRA config загружена успешно.'
''2024-12-30 18:34:51' | llmLogger | INFO     | ColQwen2ForRAG.py:96 - Процессор для извлечения загружен успешно.'
''2024-12-30 18:34:52' | llmLogger | INFO     | ColQwen2ForRAG.py:99 - Процессор для генерации загружен успешно.'
''2024-12-30 18:35:14' | llmLogger | INFO     | ColQwen2ForRAG.py:110 - Модель ColQwenLLM загружена успешно.'
''2024-12-30 18:39:40' | llmLogger | INFO     | ColQwen2ForRAG.py:88 - device = cuda:0'
''2024-12-30 18:39:40' | llmLogger | INFO     | ColQwen2ForRAG.py:93 - LoRA config загружена успешно.'
''2024-12-30 18:39:41' | llmLogger | INFO     | ColQwen2ForRAG.py:96 - Процессор для извлечения загружен успешно.'
''2024-12-30 18:39:43' | llmLogger | INFO     | ColQwen2ForRAG.py:99 - Процессор для генерации загружен успешно.'
''2024-12-30 18:40:13' | llmLogger | INFO     | ColQwen2ForRAG.py:88 - device = cuda:0'
''2024-12-30 18:40:14' | llmLogger | INFO     | ColQwen2ForRAG.py:93 - LoRA config загружена успешно.'
''2024-12-30 18:40:15' | llmLogger | INFO     | ColQwen2ForRAG.py:96 - Процессор для извлечения загружен успешно.'
''2024-12-30 18:40:17' | llmLogger | INFO     | ColQwen2ForRAG.py:99 - Процессор для генерации загружен успешно.'
''2024-12-30 18:40:34' | llmLogger | INFO     | ColQwen2ForRAG.py:110 - Модель ColQwenLLM загружена успешно.'
''2024-12-30 19:10:46' | llmLogger | INFO     | ColQwen2ForRAG.py:89 - device = cuda:0'
''2024-12-30 19:10:46' | llmLogger | INFO     | ColQwen2ForRAG.py:94 - LoRA config загружена успешно.'
''2024-12-30 19:10:48' | llmLogger | INFO     | ColQwen2ForRAG.py:97 - Процессор для извлечения загружен успешно.'
''2024-12-30 19:10:49' | llmLogger | INFO     | ColQwen2ForRAG.py:100 - Процессор для генерации загружен успешно.'
''2024-12-30 19:11:09' | llmLogger | INFO     | ColQwen2ForRAG.py:111 - Модель ColQwenLLM загружена успешно.'
''2024-12-30 19:11:29' | llmLogger | ERROR    | processor.py:113 - An error occurred during the image query process.'
Traceback (most recent call last):
  File "/opt/app-root/src/ColQwenLLM/domain/processor.py", line 105, in image_query
    prompt = conversation.format(query=query)
             ^^^^^^^^^^^^^^^^^^^
AttributeError: 'list' object has no attribute 'format'
''2024-12-30 19:12:28' | llmLogger | INFO     | ColQwen2ForRAG.py:89 - device = cuda:0'
''2024-12-30 19:12:28' | llmLogger | INFO     | ColQwen2ForRAG.py:94 - LoRA config загружена успешно.'
''2024-12-30 19:12:30' | llmLogger | INFO     | ColQwen2ForRAG.py:97 - Процессор для извлечения загружен успешно.'
''2024-12-30 19:12:31' | llmLogger | INFO     | ColQwen2ForRAG.py:100 - Процессор для генерации загружен успешно.'
''2024-12-30 19:12:49' | llmLogger | INFO     | ColQwen2ForRAG.py:111 - Модель ColQwenLLM загружена успешно.'
''2024-12-30 19:13:00' | llmLogger | ERROR    | processor.py:113 - An error occurred during the image query process.'
Traceback (most recent call last):
  File "/opt/app-root/src/ColQwenLLM/domain/processor.py", line 105, in image_query
    prompt = conversation.format(query=query)
             ^^^^^^^^^^^^^^^^^^^
AttributeError: 'dict' object has no attribute 'format'
''2024-12-30 19:16:52' | llmLogger | INFO     | ColQwen2ForRAG.py:89 - device = cuda:0'
''2024-12-30 19:16:53' | llmLogger | INFO     | ColQwen2ForRAG.py:94 - LoRA config загружена успешно.'
''2024-12-30 19:16:54' | llmLogger | INFO     | ColQwen2ForRAG.py:97 - Процессор для извлечения загружен успешно.'
''2024-12-30 19:16:56' | llmLogger | INFO     | ColQwen2ForRAG.py:100 - Процессор для генерации загружен успешно.'
''2024-12-30 19:17:14' | llmLogger | INFO     | ColQwen2ForRAG.py:111 - Модель ColQwenLLM загружена успешно.'
''2024-12-30 19:17:32' | llmLogger | INFO     | processor.py:89 - Decoding the generated response.'
''2024-12-30 19:21:08' | llmLogger | INFO     | ColQwen2ForRAG.py:89 - device = cuda:0'
''2024-12-30 19:21:08' | llmLogger | INFO     | ColQwen2ForRAG.py:94 - LoRA config загружена успешно.'
''2024-12-30 19:21:09' | llmLogger | INFO     | ColQwen2ForRAG.py:97 - Процессор для извлечения загружен успешно.'
''2024-12-30 19:21:11' | llmLogger | INFO     | ColQwen2ForRAG.py:100 - Процессор для генерации загружен успешно.'
''2024-12-30 19:21:29' | llmLogger | INFO     | ColQwen2ForRAG.py:111 - Модель ColQwenLLM загружена успешно.'
''2024-12-30 19:21:39' | llmLogger | INFO     | processor.py:89 - Decoding the generated response.'
''2024-12-30 19:29:56' | llmLogger | INFO     | ColQwen2ForRAG.py:89 - device = cuda:0'
''2024-12-30 19:29:56' | llmLogger | INFO     | ColQwen2ForRAG.py:94 - LoRA config загружена успешно.'
''2024-12-30 19:29:57' | llmLogger | INFO     | ColQwen2ForRAG.py:97 - Процессор для извлечения загружен успешно.'
''2024-12-30 19:29:59' | llmLogger | INFO     | ColQwen2ForRAG.py:100 - Процессор для генерации загружен успешно.'
''2024-12-30 19:30:18' | llmLogger | INFO     | ColQwen2ForRAG.py:111 - Модель ColQwenLLM загружена успешно.'
''2024-12-30 19:31:07' | llmLogger | ERROR    | llm_router.py:31 - An error occurred while processing image embeddings.'
Traceback (most recent call last):
  File "/opt/app-root/src/ColQwenLLM/api/llm_router.py", line 25, in image_embeddings
    img = Image.open(io.BytesIO(request_object_content))
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/PIL/Image.py", line 3498, in open
    raise UnidentifiedImageError(msg)
PIL.UnidentifiedImageError: cannot identify image file <_io.BytesIO object at 0x7f09395aeca0>
''2024-12-30 19:31:43' | llmLogger | ERROR    | llm_router.py:55 - An error occurred during the chat processing.'
Traceback (most recent call last):
  File "/opt/app-root/src/ColQwenLLM/api/llm_router.py", line 50, in image_chat
    img = Image.open(io.BytesIO(request_object_content))
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/PIL/Image.py", line 3498, in open
    raise UnidentifiedImageError(msg)
PIL.UnidentifiedImageError: cannot identify image file <_io.BytesIO object at 0x7f08fc044db0>
''2024-12-31 02:33:29' | llmLogger | INFO     | ColQwen2ForRAG.py:89 - device = cuda:0'
''2024-12-31 02:33:29' | llmLogger | INFO     | ColQwen2ForRAG.py:94 - LoRA config загружена успешно.'
''2024-12-31 02:33:31' | llmLogger | INFO     | ColQwen2ForRAG.py:97 - Процессор для извлечения загружен успешно.'
''2024-12-31 02:33:32' | llmLogger | INFO     | ColQwen2ForRAG.py:100 - Процессор для генерации загружен успешно.'
''2024-12-31 02:33:56' | llmLogger | INFO     | ColQwen2ForRAG.py:111 - Модель ColQwenLLM загружена успешно.'
''2024-12-31 02:50:16' | llmLogger | INFO     | ColQwen2ForRAG.py:89 - device = cuda:0'
''2024-12-31 02:50:17' | llmLogger | INFO     | ColQwen2ForRAG.py:94 - LoRA config загружена успешно.'
''2024-12-31 02:50:18' | llmLogger | INFO     | ColQwen2ForRAG.py:97 - Процессор для извлечения загружен успешно.'
''2024-12-31 02:50:19' | llmLogger | INFO     | ColQwen2ForRAG.py:100 - Процессор для генерации загружен успешно.'
''2024-12-31 02:50:43' | llmLogger | INFO     | ColQwen2ForRAG.py:111 - Модель ColQwenLLM загружена успешно.'
''2024-12-31 03:37:25' | llmLogger | INFO     | ColQwen2ForRAG.py:89 - device = cuda:0'
''2024-12-31 03:37:25' | llmLogger | INFO     | ColQwen2ForRAG.py:94 - LoRA config загружена успешно.'
''2024-12-31 03:37:27' | llmLogger | INFO     | ColQwen2ForRAG.py:97 - Процессор для извлечения загружен успешно.'
''2024-12-31 03:37:28' | llmLogger | INFO     | ColQwen2ForRAG.py:100 - Процессор для генерации загружен успешно.'
''2024-12-31 03:37:52' | llmLogger | INFO     | ColQwen2ForRAG.py:111 - Модель ColQwenLLM загружена успешно.'
''2024-12-31 12:51:30' | llmLogger | INFO     | ColQwen2ForRAG.py:89 - device = cuda:0'
''2024-12-31 12:51:30' | llmLogger | INFO     | ColQwen2ForRAG.py:94 - LoRA config загружена успешно.'
''2024-12-31 12:51:32' | llmLogger | INFO     | ColQwen2ForRAG.py:97 - Процессор для извлечения загружен успешно.'
''2024-12-31 12:51:33' | llmLogger | INFO     | ColQwen2ForRAG.py:100 - Процессор для генерации загружен успешно.'
''2024-12-31 12:52:01' | llmLogger | INFO     | ColQwen2ForRAG.py:111 - Модель ColQwenLLM загружена успешно.'
''2024-12-31 14:38:16' | llmLogger | INFO     | ColQwen2ForRAG.py:89 - device = cuda:0'
''2024-12-31 14:38:16' | llmLogger | INFO     | ColQwen2ForRAG.py:94 - LoRA config загружена успешно.'
''2024-12-31 14:38:18' | llmLogger | INFO     | ColQwen2ForRAG.py:97 - Процессор для извлечения загружен успешно.'
''2024-12-31 14:38:19' | llmLogger | INFO     | ColQwen2ForRAG.py:100 - Процессор для генерации загружен успешно.'
''2024-12-31 14:38:46' | llmLogger | INFO     | ColQwen2ForRAG.py:111 - Модель ColQwenLLM загружена успешно.'
''2024-12-31 14:39:10' | llmLogger | INFO     | llm_router.py:33 - Successfully processed image embeddings.'
''2024-12-31 14:40:02' | llmLogger | INFO     | llm_router.py:33 - Successfully processed image embeddings.'
''2025-01-01 08:30:56' | llmLogger | INFO     | ColQwen2ForRAG.py:87 - device = cuda:0'
''2025-01-01 08:30:56' | llmLogger | INFO     | ColQwen2ForRAG.py:92 - LoRA config загружена успешно.'
''2025-01-01 08:30:58' | llmLogger | INFO     | ColQwen2ForRAG.py:95 - Процессор для извлечения загружен успешно.'
''2025-01-01 08:30:59' | llmLogger | INFO     | ColQwen2ForRAG.py:98 - Процессор для генерации загружен успешно.'
''2025-01-01 08:31:29' | llmLogger | INFO     | ColQwen2ForRAG.py:109 - Модель ColQwenLLM загружена успешно.'
''2025-01-01 08:31:40' | llmLogger | INFO     | llm_router.py:37 - Successfully processed image embeddings.'
''2025-01-01 08:41:14' | llmLogger | INFO     | ColQwen2ForRAG.py:87 - device = cuda:0'
''2025-01-01 08:41:14' | llmLogger | INFO     | ColQwen2ForRAG.py:92 - LoRA config загружена успешно.'
''2025-01-01 08:41:16' | llmLogger | INFO     | ColQwen2ForRAG.py:95 - Процессор для извлечения загружен успешно.'
''2025-01-01 08:41:17' | llmLogger | INFO     | ColQwen2ForRAG.py:98 - Процессор для генерации загружен успешно.'
''2025-01-01 08:41:44' | llmLogger | INFO     | ColQwen2ForRAG.py:109 - Модель ColQwenLLM загружена успешно.'
''2025-01-01 08:42:02' | llmLogger | INFO     | llm_router.py:37 - Successfully processed image embeddings.'
''2025-01-01 08:52:16' | llmLogger | INFO     | ColQwen2ForRAG.py:87 - device = cuda:0'
''2025-01-01 08:52:16' | llmLogger | INFO     | ColQwen2ForRAG.py:92 - LoRA config загружена успешно.'
''2025-01-01 08:52:18' | llmLogger | INFO     | ColQwen2ForRAG.py:95 - Процессор для извлечения загружен успешно.'
''2025-01-01 08:52:19' | llmLogger | INFO     | ColQwen2ForRAG.py:98 - Процессор для генерации загружен успешно.'
''2025-01-01 08:52:45' | llmLogger | INFO     | ColQwen2ForRAG.py:109 - Модель ColQwenLLM загружена успешно.'
''2025-01-01 08:57:22' | llmLogger | INFO     | ColQwen2ForRAG.py:87 - device = cuda:0'
''2025-01-01 08:57:22' | llmLogger | INFO     | ColQwen2ForRAG.py:92 - LoRA config загружена успешно.'
''2025-01-01 08:57:24' | llmLogger | INFO     | ColQwen2ForRAG.py:95 - Процессор для извлечения загружен успешно.'
''2025-01-01 08:57:25' | llmLogger | INFO     | ColQwen2ForRAG.py:98 - Процессор для генерации загружен успешно.'
''2025-01-01 08:57:50' | llmLogger | INFO     | ColQwen2ForRAG.py:109 - Модель ColQwenLLM загружена успешно.'
''2025-01-01 08:58:05' | llmLogger | ERROR    | llm_router.py:29 - An error occurred in image_embeddings.'
Traceback (most recent call last):
  File "/opt/app-root/src/ColQwenLLM/api/llm_router.py", line 27, in wrapper
    return await func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/app-root/src/ColQwenLLM/api/llm_router.py", line 51, in image_embeddings
    res = await get_image_embeddings([image])
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: object list can't be used in 'await' expression
''2025-01-01 09:06:19' | llmLogger | INFO     | ColQwen2ForRAG.py:87 - device = cuda:0'
''2025-01-01 09:06:19' | llmLogger | INFO     | ColQwen2ForRAG.py:92 - LoRA config загружена успешно.'
''2025-01-01 09:06:21' | llmLogger | INFO     | ColQwen2ForRAG.py:95 - Процессор для извлечения загружен успешно.'
''2025-01-01 09:06:22' | llmLogger | INFO     | ColQwen2ForRAG.py:98 - Процессор для генерации загружен успешно.'
''2025-01-01 09:06:48' | llmLogger | INFO     | ColQwen2ForRAG.py:109 - Модель ColQwenLLM загружена успешно.'
''2025-01-01 09:06:55' | llmLogger | INFO     | llm_router.py:55 - Successfully processed image embeddings.'
''2025-01-01 09:07:03' | llmLogger | ERROR    | llm_router.py:41 - Invalid image file'
Traceback (most recent call last):
  File "/opt/app-root/src/ColQwenLLM/api/llm_router.py", line 38, in file_to_image
    img = Image.open(io.BytesIO(request_object_content))
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/PIL/Image.py", line 3498, in open
    raise UnidentifiedImageError(msg)
PIL.UnidentifiedImageError: cannot identify image file <_io.BytesIO object at 0x7ff0de3eb0b0>
''2025-01-01 09:16:06' | llmLogger | INFO     | ColQwen2ForRAG.py:87 - device = cuda:0'
''2025-01-01 09:16:06' | llmLogger | INFO     | ColQwen2ForRAG.py:92 - LoRA config загружена успешно.'
''2025-01-01 09:16:08' | llmLogger | INFO     | ColQwen2ForRAG.py:95 - Процессор для извлечения загружен успешно.'
''2025-01-01 09:16:09' | llmLogger | INFO     | ColQwen2ForRAG.py:98 - Процессор для генерации загружен успешно.'
''2025-01-01 09:16:37' | llmLogger | INFO     | ColQwen2ForRAG.py:109 - Модель ColQwenLLM загружена успешно.'
''2025-01-01 09:16:56' | llmLogger | INFO     | llm_router.py:70 - Successfully processed image chat.'
''2025-01-01 09:17:49' | llmLogger | INFO     | llm_router.py:70 - Successfully processed image chat.'
''2025-01-01 09:18:09' | llmLogger | INFO     | llm_router.py:55 - Successfully processed image embeddings.'
''2025-01-01 09:18:15' | llmLogger | INFO     | llm_router.py:63 - Successfully processed text embeddings.'
''2025-01-01 09:19:10' | llmLogger | INFO     | ColQwen2ForRAG.py:87 - device = cuda:0'
''2025-01-01 09:19:10' | llmLogger | INFO     | ColQwen2ForRAG.py:92 - LoRA config загружена успешно.'
''2025-01-01 09:19:12' | llmLogger | INFO     | ColQwen2ForRAG.py:95 - Процессор для извлечения загружен успешно.'
''2025-01-01 09:19:14' | llmLogger | INFO     | ColQwen2ForRAG.py:98 - Процессор для генерации загружен успешно.'
''2025-01-01 09:19:41' | llmLogger | INFO     | ColQwen2ForRAG.py:109 - Модель ColQwenLLM загружена успешно.'
''2025-01-01 09:19:45' | llmLogger | INFO     | llm_router.py:70 - Successfully processed image chat.'
''2025-01-01 09:21:35' | llmLogger | INFO     | ColQwen2ForRAG.py:87 - device = cuda:0'
''2025-01-01 09:21:36' | llmLogger | INFO     | ColQwen2ForRAG.py:92 - LoRA config загружена успешно.'
''2025-01-01 09:21:37' | llmLogger | INFO     | ColQwen2ForRAG.py:95 - Процессор для извлечения загружен успешно.'
''2025-01-01 09:21:38' | llmLogger | INFO     | ColQwen2ForRAG.py:98 - Процессор для генерации загружен успешно.'
''2025-01-01 09:22:05' | llmLogger | INFO     | ColQwen2ForRAG.py:109 - Модель ColQwenLLM загружена успешно.'
''2025-01-01 09:22:11' | llmLogger | INFO     | llm_router.py:63 - Successfully processed text embeddings.'
''2025-01-01 09:22:20' | llmLogger | INFO     | llm_router.py:70 - Successfully processed image chat.'
